src/www/ui/search.php

```diff
@@ -48,6 +48,22 @@ function PostInitialize()
     return $this->State;
   }
 
+  function loadUploads()
+  {
+    $allUploadsPre = $this->uploadDao->getActiveUploadsArray();
+    $filteredUploadsList = array();
+
+    foreach($allUploadsPre as $row)
```

Comment:
How about using `array_filter()` instead of a `foreach` loop?

Reply:
Changed